addon/components/hyper-table-v2/manage-columns.hbs

```diff
@@ -0,0 +1,52 @@
+<div class="hypertable__manage-fields" {{on-click-outside this.closeAvailableFields}}>
+  <OSS::Button @skin="primary" @icon="fas fa-columns" @label="Manage Fields" {{on "click" this.openAvailableFields}} />
+  <div class={{concat "available-fields-wrapper" (if this.displayAvailableFields " visible" " invisible")}}>
+    <div class="available-fields-wrapper__categories">
+      <div class={{concat "field-category" (unless this._activeColumnCategory " field-category--active")}}
+           role="button" {{on "click" (fn this.setFieldCategory null)}} data-control-name="field_category_toggle_all_fields">
+        <div>{{t "hypertable.features.manage_fields.all_fields"}}</div>
+        <div><i class="fas fa-caret-right"></i></div>
+      </div>
+      {{#each this._columnCategories as |category|}}
+        <div class={{concat "field-category" (if (eq this._activeColumnCategory category) " field-category--active")}}
+             role="button" data-control-name={{concat "field_category_toggle_" category}}
+             {{on "click" (fn this.setFieldCategory category)}}>
+          <div>{{category}}</div>
+          <div><i class="fa fa-caret-right"></i></div>
+        </div>
+      {{/each}}
+    </div>
+    <div class="available-fields-wrapper__fields">
+      <div class="search">
+        <Input
+          @type="text" @value={{this._searchColumnDefinitionKeyword}}
+          data-control-name="column_definitions_search_input" class="upf-input upf-input--small"
+          placeholder={{t "hypertable.features.manage_fields.search_placeholder"}}
+        />
+      </div>
+      <div class="fields-list">
+        {{#each-in this._orderedFilteredClusters as |clusterName fields|}}
+          {{#if clusterName}}
+            <p class="cluster-name text-style-bold margin-top-xxx-sm margin-bottom-xxx-sm">
```

Comment:
Any particular reason why this is a `<p>` tag ? 🤔

Reply:
No, I just use the old version, I use div now